test/distributed/optim/test_zero_redundancy_optimizer.py

```diff
@@ -26,11 +26,23 @@
     allreduce_hook,
 )
 from torch.distributed.algorithms.join import _Join, _Joinable, _JoinHook
-from torch.distributed.optim import ZeroRedundancyOptimizer
-from torch.distributed.optim.zero_redundancy_optimizer import _broadcast_object
+from torch.distributed.zero import ZeroRedundancyOptimizer
```

Comment:
> In addition, keep backwards compatibility by still being able to use from torch.distributed.optim import ZeroRedundancyOptimizer as the previous import was.

Would I be correct if I assume the original imports should still work? If so, shall we drop this change?